### DISCUSSION TITLE
Just Wanted to Say Thank You

### DISCUSSION THREAD

**Discussion by bongochong** [SebastJava/mint-themes#56] [General]
This is stellar theming work. Far more consistent and polished than the upstream Mint-Y themes (in their current state at least), and the elimination of rounded corners for the vast majority of application windows is really nice. I've been on Fedora's Cinnamon spin for around 3 years now, and we get major upgrades to Cinnamon every so often. The jump from 5.2.x to 5.6.x brought us the in-progress revamps of the Mint-Y and Mint-X themes, which are distractingly unfinished. Your Mint-Yz themes are a breath of fresh air.

**Comment by SebastJava** [comment]
Thanks. This helps me to keep moving forward. I am currently having some down time. I had bad feedback on my latest project and this brings bad moods to me. I wanted to show you that bad adventure on the LinuxMint forums, but i can't find the link because... the forums servers are down again ! Anyway, it's about this Prosilver-lm, in the top first topics, in the **Suggestions and feedback** forum on https://forums.linuxmint.com/. Well, i got bad feedback there. They prefer their standard forums theme over my https://sebastjava.com/forums/. That's not the kind of thing that keeps me moving on.

Also, i had other issues, in the real material world, outside of this virtual world here. I will have to stop and think about my social and economic condition, and see if i could get rid of my dirty-messy neighbors by moving to some upper area. I am not sure about that, but i will try to dream it...

I was thinking about this Prosilver-lm again, and i can understand some critics, to some extents. Short story, i was trying to get some approvals by using approximately the same color palette as the one found on the [LinuxMint's main website](https://linuxmint.com/). But that's not the best choice to get good contrasts and eye comfort. I find it is much easier to go toward higher hues, somewhere in the blue tones. But then i bet i will be told this is out of the standard Mint's color range. I always found their sort of grayish yellow-green wasn't so minty anyway.

To sum up, i am currently trying to untangle myself and get this Prosilver-lm project finally done, or just aborted, because i am not sure there is any chance of getting any official approval anyway. And then, next, i will have to take a break and just breath some fresh air in April. And think about my broader issues.

And then, maybe, if i can't find any better place to fit in the socioeconomic game, i will be back working on these GTK themes. But probably not this Mint-Yz. This Mint-Yz might end up being archived, and i would start working on something new: Mint-Z.

**Comment by bongochong** [comment]
I think the themes are basically perfect, and in regard to personal + socioeconomic struggles, most of us have been there, myself included (almost 3 months ago the closest person in the world to me died, and I don't know when / if I'll feel completely *back to myself* again). Do what brings you joy whenever you can, and definitely try to ignore useless criticism. You do great work, and it saved me countless days of fruitlessly tweaking GTK CSS files until my eyes bled. I'm sure you saved others a lot of time and frustration as well. Godspeed and don't give up!
